### DISCUSSION TITLE
无法在阅读模式预览笔记，只能在编辑模式下看到笔记内容

### DISCUSSION THREAD

**Discussion by JohnWhipples** [tamlok/vnote#2226] [Q&A] [answered]
如图，debian系统下appiamge版本的VNote突然无法在阅读模式下预览笔记，只能在编辑模式下看到笔记的内容，3.13和最新的3.15版本问题都相同，win10下则没有该问题。请问大家是否遇到类似问题？

![图片](https://user-images.githubusercontent.com/58494778/187033599-ec74225e-2a48-421f-9888-c262b7174a3e.png)
![图片](https://user-images.githubusercontent.com/58494778/187033642-894725bf-d7e7-4306-ab50-11e705ed6fe5.png)

**Comment by tamlok** [comment, marked as answer]
加参数 `--no-sandbox` 运行即可

**Reply by JohnWhipples** [reply]
已解决，感谢！

**Reply by KINOonMars** [reply]
在哪里加呢，也是这个问题

**Reply by KINOonMars** [reply]
> 在哪里加呢，也是这个问题

但是是win x64,v3.17.0

**Reply by tamlok** [reply]
> > 在哪里加呢，也是这个问题
> 
> 但是是win x64,v3.17.0

Windows 用户请看这个置顶Issue，一般是OpenGL的问题：

https://github.com/vnotex/vnote/issues/853

**Reply by KINOonMars** [reply]
> > > 在哪里加呢，也是这个问题
> > 
> > 
> > 但是是win x64,v3.17.0
> 
> Windows 用户请看这个置顶Issue，一般是OpenGL的问题：
> 
> #853

都试了，还是不行

**Reply by tamlok** [reply]
> > > > 在哪里加呢，也是这个问题
> > > 
> > > 
> > > 但是是win x64,v3.17.0
> > 
> > 
> > Windows 用户请看这个置顶Issue，一般是OpenGL的问题：
> > #853
> 
> 都试了，还是不行

那就用管理员模式运行看看。如果是只是什么都不显示，但vnote本身没有卡死，那应该是浏览器进程起不来。

**Reply by KINOonMars** [reply]
> > > > > 在哪里加呢，也是这个问题
> > > > 
> > > > 
> > > > 但是是win x64,v3.17.0
> > > 
> > > 
> > > Windows 用户请看这个置顶Issue，一般是OpenGL的问题：
> > > #853
> > 
> > 
> > 都试了，还是不行
> 
> 那就用管理员模式运行看看。如果是只是什么都不显示，但vnote本身没有卡死，那应该是浏览器进程起不来。

是这样的，启用管理员后，开阅读模式还是空白，有办法解决吗？

**Reply by tamlok** [reply]
打开一个文件，阅读模式，按F12调试，Console里面看看有没有错误信息？或者右上角，查看日志，看看有没有相关error。

**Reply by KINOonMars** [reply]
> 打开一个文件，阅读模式，按F12调试，Console里面看看有没有错误信息？或者右上角，查看日志，看看有没有相关error。
```
Warning:(theme.cpp:398) undefined or invalid palette color 
"widgets#mainwindow#side_bar#icon#selected#fg"
Warning:(:0) No language dictionaries for the language: "zh_CN" trying to load en_US as default
```
日志里是这样的

**Reply by KINOonMars** [reply]
> 打开一个文件，阅读模式，按F12调试，Console里面看看有没有错误信息？或者右上角，查看日志，看看有没有相关error。

还有一个

```
Info:(main.cpp:149) OpenSSL build version: "OpenSSL 1.1.1g  21 Apr 2020" link version: "OpenSSL 1.1.1j  16 Feb 2021"
Warning:(main.cpp:153) versions of the built and linked OpenSSL mismatch, network may not work
```

**Reply by tamlok** [reply]
你打开vvnote，打开一个文件，阅读模式有问题，这时候查看日志，再贴上来看看？

**Reply by KINOonMars** [reply]
> 你打开vvnote，打开一个文件，阅读模式有问题，这时候查看日志，再贴上来看看？

```
Info:(main.cpp:144) "VNote (v3.17.0) started at 周五 6月 21 11:18:31 2024 (windows)"
Info:(main.cpp:149) OpenSSL build version: "OpenSSL 1.1.1g  21 Apr 2020" link version: "OpenSSL 1.1.1j  16 Feb 2021"
Warning:(main.cpp:153) versions of the built and linked OpenSSL mismatch, network may not work
Info:(main.cpp:209) locale: "zh_CN"
Info:(thememgr.cpp:98) using theme "moonlight"
Warning:(theme.cpp:398) undefined or invalid palette color "widgets#mainwindow#side_bar#icon#fg"
Warning:(theme.cpp:398) undefined or invalid palette color "widgets#mainwindow#side_bar#icon#selected#fg"
Warning:(:0) No language dictionaries for the language: "zh_CN" trying to load en_US as default
```

**Reply by tamlok** [reply]
按F12，然后切换到Console看看有没有什么错误？

**Reply by KINOonMars** [reply]
> 按F12，然后切换到Console看看有没有什么错误？

闪一下，然后空白

**Reply by KINOonMars** [reply]
> > 按F12，然后切换到Console看看有没有什么错误？
> 
> 闪一下，然后空白
![屏幕截图 2024-06-21 144216](https://github.com/vnotex/vnote/assets/168267020/f6120c7c-25f4-46b8-b5fe-95203b9e0c3f)
这样

**Reply by tamlok** [reply]
比较奇怪，加Q群487756074看一下吧

**Reply by KINOonMars** [reply]
> 比较奇怪，加Q群487756074看一下吧

好的

**Reply by tamlok** [reply]
结论是： VNote可执行文件文件夹要放在C盘下面。。。

**Reply by KINOonMars** [reply]
😂
